test/functional/board-layout/layout.test.ts

```diff
@@ -27,8 +27,8 @@ test(
     async (page) => {
       await page.mouseDown(boardWrapper.findItemById("A").findDragHandle().toSelector());
       await expect(page.getGrid()).resolves.toEqual([
-        ["A", "B", "C", "D"],
-        ["A", "B", "C", "D"],
+        [" ", "B", "C", "D"],
+        [" ", "B", "C", "D"],
         ["E", "B", "G", "H"],
         ["E", "B", "G", "H"],
         [" ", " ", " ", " "],
```

Comment:
Because the dragged node is now in portal, it is not displayed on the grid. The updated assertion even better reflects the actual state of the UI